scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -55,6 +55,18 @@
   }
 }
 
+.nav-pills {
+  margin-bottom: 15px;
```

Comment:
you might consider that a consistent look throughout civihr, not shoreditch. bootstrap does not impose a bottom margin on the nav pills, then as a rule shoreditch shouldn't either